cmd/daprd/main.go

```diff
@@ -151,6 +151,12 @@ import (
 
 	bindings_loader "github.com/dapr/dapr/pkg/components/bindings"
 
+	// Workflows.
+	"github.com/dapr/components-contrib/workflows"
+	temporal "github.com/dapr/components-contrib/workflows/temporal"
```

Comment:
Is the `temporal` alias needed here or can it be removed (for consistency with the other imports?
```suggestion
	"github.com/dapr/components-contrib/workflows/temporal"
```